### ISSUE TITLE
[BUG] Search has been broken since Kavita 0.8

### ISSUE THREAD

**Issue by KaitoKid** [ACK72/kavya-paperback#18]
**Describe the bug**
Any searches cause a Paperback bug. I believe it started w/ the latest update but I'm not sure - unfortunately it's a bit difficult to rollback Kavita to 0.7.x 

`Paperback: JSError Syntax Error [PaperbackError(300) JS Error SyntaxError: JSON Parse error: Unxpected EOF - parse@[native code]@:1255:34`

```
[Kavita] [2024-04-18 22:50:57.531 -07:00  88] [Debug] API.Controllers.LibraryController Caching libraries for library_pomona
[Kavita] [2024-04-18 22:50:57.531 -07:00  88] [Information] Serilog.AspNetCore.RequestLoggingMiddleware HTTP GET /api/Library responded 204 in 3.2305 ms
```

**Comment by KaitoKid** [comment]
Fixed in #19

**Comment by ACK72** [comment]
Thank you so much for identifying the issue and writing a fix commit. I was going to fix it this weekend because I was short on time, but you've saved me some time. Again, thank you so much.
